app/enterprise/0.32-x/upgrades-migrations.md

```diff
@@ -0,0 +1,55 @@
+---
+title: Upgrades and Migrations
+---
+
+# Upgrades and Migrations
+## Upgrades of Kong EE
+
+We define an upgrade, the process of changing the version of the software currently deployed to a more recent one, there is no difference with concept of update; normally associated with an upgrade between minor versions.
+
+We suggest to follow the below steps to perform an effective upgrade, this approach will require downtime, in case you need high availability review the Blue/Green Deployment section.
+
+1. Download the destination version of the product you are interest to upgrade to in Bintray
+2. Take a backup copy of the production database
+3. Review the [changelog](https://docs.konghq.com/enterprise/changelog/) for the destination version of the upgrade
+4. Prepare the preproduction environment with the copy of the database taken from production
+5. Install the newer package in preproduction.
+6. Apply any breaking changes to your preproduction environment
+7. Run the command kong migrations up
```

Comment:
"Run the command `kong migrations up`" - the command should be within a code block